spec/requests/api/v0/university_profile_retrieves_keys_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe 'University Profile retrieves research keys', type: :request do
+  let(:university) { create(:user, role: :university, subscriber: true) }
+  let(:credentials) { university.create_new_auth_token }
+  let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(credentials) }
+
+  describe 'research keys are returned for subscribed user' do
+    before do
+      5.times { RegistrationKey.create(user: university) }
+      get "/api/v0/users/#{university.id}", headers: headers, params: { id: university.id }
```

Comment:
You can remove the params

Reply:
Right, because all the info we need is already in the headers.  Done.